index.bs

```diff
@@ -277,12 +277,14 @@ partial interface Document {
 </pre>
 
 The {{pictureInPictureEnabled}} attribute's getter must return
-<code>true</code> if the <a>context object</a> is <a>allowed to use</a> the
-feature indicated by attribute name <code>picture-in-picture</code> and
-<a>Picture-in-Picture is supported</a>, and false otherwise.
-
-<dfn>Picture-in-Picture is supported</dfn> if there is no
-previously-established user preference, security risk, or platform limitation.
+<code>true</code> if <a>Picture-in-Picture support</a> is <code>true</code> and
+the <a>context object</a> is <a>allowed to use</a> the feature indicated by
+attribute name <code>picture-in-picture</code>, and <code>false</code>
+otherwise.
+
+<dfn>Picture-in-Picture support</dfn> is <code>true</code> if there is no
+previously-established user preference, security risk, or platform limitation,
```

Comment:
security risk? That's unclear. Restrictions? Though maybe keep to "platform limitation"?